src/main/java/seedu/address/model/commons/WanderlustDate.java

```diff
@@ -70,12 +71,51 @@ public boolean isAfter(WanderlustDate toCompare) {
         return value.isAfter(toCompare.getValue());
     }
 
+    /**
+     * Returns true if the start date is before or on the same day as end date.
+     */
+    public static boolean isValidStartAndEndDate(WanderlustDate startDate, WanderlustDate endDate) {
+        return startDate.getValue().compareTo(endDate.getValue()) <= 0;
+    }
 
+    /**
+     * Returns true if the date is set on today's date or after today's date.
+     */
+    public static boolean isDateAfterToday(WanderlustDate startDate) {
```

Comment:
I think you can remove this method? Since we will not be using it anymore.

Reply:
ok!